### ISSUE TITLE
Generic.NamingConventions.ConstructorName complains about old constructor in interfaces

### ISSUE THREAD

**Issue by mimmi20** [squizlabs/PHP_CodeSniffer#2663]
In the [Phing project](https://github.com/phingofficial/phing) there is this Interface

```php
interface CustomChildCreator
{
    /**
     * Creates the object for the child element
     *
     * @param  string $elementName the name of the element that has been requested
     * @param  Project $project The project the element is in
     * @return object  Returns the nested element
     */
    public function customChildCreator($elementName, Project $project);
}
```

With the CodeSniffer I got this error:

```shell
FILE: ...cuments\GitHub\phing\classes\phing\parser\CustomChildCreator.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 36 | ERROR | PHP4 style constructors are not allowed; use
    |       | "__construct()" instead
    |       | (Generic.NamingConventions.ConstructorName.OldStyle)
----------------------------------------------------------------------
```

The Sniff should not compain this on Interfaces.

**Comment by gsherwood** [comment]
Thanks for reporting this. The fix will be in 3.5.2
